index.html

```diff
@@ -32,32 +59,43 @@ <h3 class="SD">I'm a Software Developer</h3>
           <a href="https://twitter.com/OfficialAseT"><img src="/images/twitter.png" class="contact-icon"></a>
           <a href="#"><img src="/images/medium.png" class="contact-icon"></a>
         </section>
-    </div>
+  </div>
 </section>
 
     <section class="Recent-works">
-
+      <section class="recent-works-heading">
         <h1 class="recent-works-title">My Recent Works</h1>
-      
+        <div class="line"></div>
```

Comment:
I have observed a few design issues, that could be eliminated using `dislay: none` when you transition from the mobile to the desktop version. To look at, how you can do that, you can follow [this guide.](https://www.w3schools.com/css/css_display_visibility.asp) 

1. So this line:
![image](https://user-images.githubusercontent.com/61361037/213734331-f132bf8d-d887-47b9-8793-4d8714335fbd.png)

is appearing in the desktop version here:
![image](https://user-images.githubusercontent.com/61361037/213734507-1b88a2a4-c9d0-49ff-8848-6923328283a4.png)

2. This line
![image](https://user-images.githubusercontent.com/61361037/213734770-2617b68a-7e58-4f42-a79b-92445f08bba5.png)

in the desktop version appears like this
![image](https://user-images.githubusercontent.com/61361037/213735006-f756e817-07bd-4f94-a40c-f02746d498be.png)

But keep in mind, here in the `form` section, you do not have to use the `display: none` property, you have to switch from the small bold line to this
![image](https://user-images.githubusercontent.com/61361037/213744257-17d74a12-80db-40a9-bc0b-69120907f403.png)